Quaver.Shared/QuaverGame.cs

```diff
@@ -433,5 +442,51 @@ private void HandleKeyPressCtrlO()
                     break;
             }
         }
+
+        /// <summary>
+        ///    Handles when the user holds Control, Shift and Alt, and presses R
+        /// </summary>
+        private void HandleKeyPressCtrlShiftAltR()
+        {
+            // Check for modifier keys
+            if (!((KeyboardManager.CurrentState.IsKeyDown(Keys.LeftControl) || KeyboardManager.CurrentState.IsKeyDown(Keys.RightControl)) &&
+                  (KeyboardManager.CurrentState.IsKeyDown(Keys.LeftShift) || KeyboardManager.CurrentState.IsKeyDown(Keys.RightShift)) &&
+                  (KeyboardManager.CurrentState.IsKeyDown(Keys.LeftAlt) || KeyboardManager.CurrentState.IsKeyDown(Keys.RightAlt))))
+                return;
+            
+            if (!KeyboardManager.IsUniqueKeyPress(Keys.R))
```

Comment:
Pretty sure the keybind in osu! S, not R. That should preferably carry over. I thought it wasn't working because the keybind was different 😛

Reply:
Well, I didn't want to directly copy the same keybind, but i'll change it up if you really want.

Reply:
Might as well. It's just a bit weird since you have to use two hands to reload it (at least for me).

Reply:
two hands? i only need my pinky, thumb, ring finger and index finger to click the combination